### PR TITLE
NIDaq instance only launches on editingFinished event from spinbox or…

### PR DESCRIPTION
**Pull request by ljjh20**
… onMouseReleaseEvent of slider. Temporarily reimplemented placeholder thread that doesn't crash without Bin hardware for testing

### CODE REVIEW

File: widgets/gui/qt_nidaq_worker.py [hunk 1]
```diff
@@ -16,30 +17,37 @@ def __init__(self, parameters, view, channel, *args, **kwargs):
...
+        #                      offset_view2=self.parameters[7],
+        #                      view1_galvo1=self.parameters[8],
+        #                      view1_galvo2=self.parameters[9],
+        #                      view2_galvo1=self.parameters[10],
+        #                      view2_galvo2=self.parameters[11],
+        #                      stripe_reduction_range=self.parameters[12],
+        #                      stripe_reduction_offset=self.parameters[13])
+        #
+        #self.daq_card.select_view(1)
+        #self.daq_card.select_channel_remove_stripes(488)
```
Comment (AhmetCanSolak) [inline widgets/gui/qt_nidaq_worker.py:49]: do not do this please, we will test it on Bin's machine. Once you have something ready to test on Bin's machine let me know.

**Review by AhmetCanSolak — CHANGES_REQUESTED** [review]

File: widgets/gui/qt_nidaq_worker.py (hunk 1 above)
Comment (ljjh20) [reply widgets/gui/qt_nidaq_worker.py:49]: Ok I will change it

Comment (AhmetCanSolak) [reply widgets/gui/qt_nidaq_worker.py:49]: thank you @ljjh20 !

**Comment by ljjh20** [comment]
Branch is updated. It is forked from the master you pushed today, so no disabled parameters and uses default inputs for view and channel, instead of combobox selections (fixed in live_function_parameters branch)

File: widgets/gui/qt_live_control.py [hunk 2]
```diff
@@ -45,37 +49,51 @@ def __init__(self, parent, button_name):
...
-            daq_card_thread = qt_nidaq_worker.NIDaqWorker(parameters, view, channel)
-            # connect
-            self.trigger_stop_live.connect(daq_card_thread.stop)
+        while True:
+            print("waiting", self.wait_shutdown)
+            time.sleep(1)
+            # time.sleep(2)
+            if not self.wait_shutdown:
+                break
+            QApplication.processEvents()
```
Comment (AhmetCanSolak) [inline widgets/gui/qt_live_control.py:65]: this is the line that does the trick, check my changes @ljjh20  and let me know if there are any unclear part

**Pull request merged**